store/conf/attrs/zimbra-attrs.xml

```diff
@@ -9381,11 +9381,7 @@ TODO: delete them permanently from here
   <desc>Whether to enable zimbra network new generation mobile sync module.</desc>
 </attr>
 
-<attr id="2119" name="zimbraJWTKey" type="ostring" max="128" immutable="1" cardinality="single" optionalIn="account"  flags="domainAdminModifiable" since="8.8.0">
-  <desc>JWT secret key</desc>
-</attr>
-
-<attr id="2120" name="zimbraJWTLifetime" type="duration"  max="3600" cardinality="single" optionalIn="account,cos" flags="accountInherited,domainAdminModifiable" callback="PositiveTimeInterval" since="8.8.0">
+<attr id="2119" name="zimbraJWTLifetime" type="duration"  max="3600" cardinality="single" optionalIn="account,cos" flags="accountInherited,domainAdminModifiable" callback="PositiveTimeInterval" since="8.8.0">
```

Comment:
session timeout is not controlled by jwt life time. the client might request for new jwt just before the jwt is about to expire. in this way session time would still be 30 mins however jwt lifetime might only be 5 mins or so.